.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,46 +1,25 @@
 <!--
-* New contributors are highly encouraged to read our
-  [CONTRIBUTING](/CONTRIBUTING.md) documentation.
+* Contributors are encouraged to read our [CONTRIBUTING](/CONTRIBUTING.md) documentation.
 * Both Contributor and Reviewer Checklists are available at https://datadoghq.dev/datadog-agent/guidelines/contributing/#pull-requests.
 * The pull request:
   * Should only fix one issue or add one feature at a time.
   * Must update the test suite for the relevant functionality.
   * Should pass all status checks before being reviewed or merged.
 * Commit titles should be prefixed with general area of pull request's change.
+* Please fill below sections if possible with any relevant information or link.
 -->
 ### What does this PR do?
 
-<!--
-* A brief description of the change being made with this pull request.
-* If the description here cannot be expressed in a succinct form, consider
-  opening multiple pull requests instead of a single one.
--->
-
 ### Motivation
 
-<!--
-* What inspired you to submit this pull request?
-* Link any related GitHub issues or PRs here.
--->
+### Describe how to test/QA your changes
```

Comment:
I feel like having QA instructions at the end was convenient as reviewers don't need to check them, while they do want to look at the additional notes, the new layout means you have to ignore the middle of the description when reviewing (and similarly for QA-ers, you usually mostly look the QA section, so now you need to find it in the middle of the description)

Reply:
I would probably argue that the way you test things are quite valuable for a reviewer as well:
- as we want to enrich our test coverage, reviewer can encourage tests
- it's an opportunity to try assessing if tests are relevant.